fvm/evm/events/events.go

```diff
@@ -152,6 +154,7 @@ type BlockEventPayload struct {
 	ParentBlockHash     gethCommon.Hash `cadence:"parentHash"`
 	ReceiptRoot         gethCommon.Hash `cadence:"receiptRoot"`
 	TransactionHashRoot gethCommon.Hash `cadence:"transactionHashRoot"`
+	Prevrandao          gethCommon.Hash `cadence:"prevrandao"`
```

Comment:
NIT: Since this is Flow specific naming, maybe we should name it random since it's not really previous randao of the block.

Reply:
I'm worried if use Random as the name, it might convey the feeling that this is safe to use as a random number.